package.json

```diff
@@ -38,18 +38,19 @@
     "@commitlint/config-angular": "8.2.0",
     "@commitlint/prompt": "8.2.0",
     "@commitlint/prompt-cli": "8.2.0",
-    "@edx/frontend-build": "^2.0.4",
-    "@edx/frontend-platform": "1.1.14",
-    "@edx/paragon": "7.1.5",
+    "@edx/brand": "npm:@edx/brand-edx.org@1.2.0",
```

Comment:
Can we upgrade the example to use the latest 1.2.1 (it changes the style of links)